website/pages/docs/concepts/resource-quotas.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Resource Quotas
+sidebar_title: Resource Quotas
+description: Providing measures against misbehaving applications and users overdrawing resources in Vault.
+---
+
+# Resource Quotas
+
+Every single interaction with Vault, whether to put secrets into the key/value
+store or to generate new pairs of database credentials for the MySQL database,
+needs to go through Vault’s API.
+
+One side effect of the API driven model is that applications and users can
+misbehave by overwhelming system resources through consistent and high volume API
+requests resulting in DoS issues in some Vault nodes or even the entire Vault
+cluster. This risk is significantly increased when Vault API endpoints are exposed
+to thousands or millions of services deployed across a global infrastructure,
+especially in use cases of Vault where Vault is deployed as a service for internal
+developers.
+
+Vault provides a feature, resource quotas, that allows Vault operators to specify
+limits on resources used in Vault. Specifically, Vault allows operators to create
+and configure API rate limits.
+
+## Rate Limit Quotas
+
+Vault allows operators to create rate limit quotas which enforce API rate
+limiting using a [token bucket](https://en.wikipedia.org/wiki/Token_bucket) algorithm.
+A rate limit quota can be created at the root level or defined on a namespace or
+mount by specifying a `path` when creating the quota. The rate limiter is applied
+to each unique client IP address on a per-node basis (i.e. rate limit quotas
+are not replicated). A client may invoke `burst` requests at any given second,
+after which they may invoke additional requests at `rate` per-second, where `burst`
+must be greater than or equal to `rate`.
+
+A rate limit quota defined at the root level, essentially to the entire Vault API,
+is inherited to all mounts. However, a rate limit quota can be defined for a mount,
```

Comment:
"inherited to" => "inherited by"

or maybe "applied to"?

There are two ways of interpreting this, one is "all the mounts share the same token bucket" and the other is "all the mounts get a quota matching that applied to the root".  Should we clarify which is meant by this sentence?

Reply:
Updated.